static/bidder-info/mgidX.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://us-east-x.mgid.com/pserver"
+endpoint: "https://{{.Host}}.mgid.com/pserver"
```

Comment:
This change is not recommended. We highly discourage folks to use dynamic domain this way. Please review - https://github.com/prebid/prebid-server/issues/2606

Reply:
@xmgiddev requesting to address above comment